fastestimator/op/tensorop/model/update.py

```diff
@@ -38,6 +38,7 @@ def __init__(self,
                  mode: Union[None, str, Iterable[str]] = "train"):
         super().__init__(inputs=loss_name, outputs=None, mode=mode)
         self.model = model
+        self.loss_name = loss_name
```

Comment:
this should be a setter property to influence the value at inputs[0] rather than a true property. if we modify loss_name somewhere it won't be given the correct input like this

Reply:
we don't need loss_name here now.